website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -86,7 +86,21 @@ If you get the error `invalid ownership metadata; label validation error:` while
 WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE THAT YOU COPY THE BETA API ALERT SECTION FROM THE LAST RELEASE TO PROPERLY WARN USERS OF THE RISK OF UPGRADING WITHOUT GOING TO 0.32.x FIRST
 -->
 
-### Upgrading to `1.5.2`+
+### Upgrading to `1.6.0`+
+
+{{% alert title="Warning" color="warning" %}}
+Karpenter `1.1.0` drops the support for `v1beta1` APIs.
+**Do not** upgrade to `1.1.0`+ without following the [Migration Guide]({{<ref "../../v1.0/upgrading/v1-migration.md#before-upgrading-to-v110">}}).
+{{% /alert %}}
+
+* Native ODCR support has graduated to beta and is enabled by default.
```

Comment:
I am guessing this is something that needs to be updated as part of the release to include other features?

Reply:
Yeah, I'm not aware of any other breaking changes but they should be added in the docs PR for the respective feature.